api/kyverno/v2/cleanup_policy_types.go

```diff
@@ -48,6 +48,8 @@ type CleanupPolicy struct {
 	// Status contains policy runtime data.
 	// +optional
 	Status CleanupPolicyStatus `json:"status,omitempty"`
+	// DeletionPropagationPolicy defines the deletion policy for the resources
```

Comment:
This should go in the spec.

Reply:
In this ClusterCleanupPolicyList struct ?

Reply:
Only for this one of for every other files?